src/vary.cls.php

```diff
@@ -754,6 +754,9 @@ private function _cookie($val = false, $expire = false, $path = false)
 		 */
 		$is_ssl = $this->conf(Base::O_UTIL_NO_HTTPS_VARY) ? false : is_ssl();
 
-		setcookie(self::$_vary_name, $val, $expire, $path ?: COOKIEPATH, COOKIE_DOMAIN, $is_ssl, true);
+		// Fix https://wordpress.org/support/topic/fatal-error-in-litespeed-cache-setcookie-path-issue/
+		$encoded_path = urlencode( $path ?: COOKIEPATH, COOKIE_DOMAIN );
```

Comment:
```suggestion
		$encoded_path = urlencode( $path ?: COOKIEPATH);
```
`urlencode` has only one parameter.
https://www.php.net/manual/en/function.urlencode.php